versions-4.3.x.cfg

```diff
@@ -1,4 +1,13 @@
 [buildout]
+# FIXME: there are pending issues on all versions after 4.3.11
```

Comment:
I think it would be interesting to put a reference to: https://github.com/plone/plone.app.testing/issues/39